src/core/lib/gprpp/stat.h

```diff
@@ -33,6 +33,11 @@ namespace grpc_core {
 // StatusCode::kInternal will be returned.
 absl::Status GetFileModificationTime(const char* filename, time_t* timestamp);
 
+// A simple helper function that will combine |path1| and |path2| with the file
+// separator, and return the new path. It doesn't check the validity of the new
+// path.
+std::string PathJoin(std::string path1, std::string path2);
```

Comment:
The parameters here should both be `absl::string_view`, since this function does not need to take ownership.